java/org/apache/coyote/http11/Constants.java

```diff
@@ -106,7 +106,7 @@ public final class Constants {
     public static final String KEEP_ALIVE_HEADER_VALUE_TOKEN = "keep-alive";
     public static final String CHUNKED = "chunked";
     public static final byte[] ACK_BYTES = ByteChunk.convertToBytes("HTTP/1.1 100 " + CRLF + CRLF);
-    public static final String TRANSFERENCODING = "Transfer-Encoding";
+    public static final String TRANSFER_ENCODING = "Transfer-Encoding";
```

Comment:
This change still needs to be reverted. It is a public constant and my previous comments in this PR about not changing public constants apply equally to this one.